spec/github_url_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+describe 'URL Building' do
+  it 'builds a URL with the closed flag when present' do
+    url = IssueExporting.make_url 'herp', 'derp', 'TOKEN', { include_closed_issues: true }
+    expect(url).to eq "https://api.github.com/repos/herp/derp/issues?access_token=TOKEN&state=all"
+  end
+
+  it 'builds a URL without the closed flag when it is false' do
+    url = IssueExporting.make_url 'herp', 'derp', 'TOKEN', { include_closed_issues: false }
```

Comment:
Line is too long. [91/80]<br>Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.